src/components/call-tool.js

```diff
@@ -96,7 +113,33 @@ module.exports = React.createClass({
       </section>
     );
   },
+  isAfterOct29: function() {
+    var date = new Date();
+    date.setUTCHours(date.getUTCHours() + 2);
+    var day = date.getUTCDate();
+    var month = date.getUTCMonth();
+    if (month >= 9 && day >= 29) {
+      return true;
+    }
+    return false;
+  },
+  renderClosedSign: function() {
+    var hoursString = "business_hours_cest";
+    if (this.isAfterOct29()) {
+      hoursString = "business_hours_cet";
+    }
+    return (
+      <section>
+        <h2>
+          {this.context.intl.formatMessage({id: 'closed_for_business'})}
+          <br/>
```

Comment:
in the other template we’re using `{" "}`, intended?

Reply:
Yeah, in the other one we didn't want a line break, in this one it was requested to have a line break.

Reply:
sounds good!